### ISSUE TITLE
Consider a fail for generic <div> elements with aria-label, labelledby, describedby

### ISSUE THREAD

**Issue by DavidMacDonald** [dequelabs/axe-core#1904]
Consider a fail for generic <div> elements with aria-label, for instance a background image 
```<div ... aria-label=alt text ...> without role="img"```

Usually div won't announce an aria-label or aria-labelledby, or aria describedby...

I have a page of testing here
http://www.davidmacd.com/blog/does-aria-label-override-static-text.html

**Comment by straker** [comment]
Similar to #1575 (just linking them together so I remember).

**Comment by WilcoFiers** [comment]
I'm going to close this, we don't need two issues on the same topic.
